apps/artemis-web/libs/graphql.ts

```diff
@@ -408,16 +408,18 @@ export class QueryGenerator {
           condition +
           `{ ${dateField}: {_gte: "${this.options.dateRange.dateFrom}"} },{ ${dateField}: {_lte: "${this.options.dateRange.dateTo}"} },`;
       if (this.options.hasColumnFilter) {
-        let column = Object.keys(this.options.columnFilter)[0];
-        const filterValue = this.options.columnFilter[column].filterVal;
-        if (column.includes('original'))
-          column = column.replace('_original', '');
-        else if (column === 'as_path2') column = 'as_path';
-
-        if (column === 'service') {
-          condition = condition + `{ ${column}: {_like: "%${filterValue}%"} }`;
-        } else if (column !== 'as_path')
-          condition = condition + `{ ${column}: {_eq: "${filterValue}"} }`;
+        Object.keys(this.options.columnFilter).forEach((column) => {
+          const filterValue = this.options.columnFilter[column].filterVal;
+          if (column.includes('original'))
+            column = column.replace('_original', '');
+          else if (column === 'as_path2') column = 'as_path';
```

Comment:
why is this 'as_path2' used exact;y, and why not directly use 'as_path'?

Reply:
because as_path2 is a modified version of as_path, with added tooltips, I could also change this to use the as_path name, its no prob

Reply:
OK then, probably it is fine, I was just worried that it was forgotten from mock naming. Resolving this comment.